### ISSUE TITLE
Investigate AWS IoT Framework for controlling Fetch

### ISSUE THREAD

**Issue by zephirefaith** [CogRob/workspace#35]
I remember the Amazon engineers talking about their IoT resources, which basically creates an endpoint for any cloud-connected device and provides stable REST APIs. Before solidifying our end of the architecture and making commitments to any one particular framework let's give this a go:
https://developer.amazon.com/public/community/post/Tx3828JHC7O9GZ9/Using-Alexa-Skills-Kit-and-AWS-IoT-to-Voice-Control-Connected-Devices

It basically does what Henrik wanted to do initially, except this is way more flexible than the Smart Home Alexa API. This will also support having all communication on Raspberry Pi which locally connects to Fetch, as Henrik mentioned last week.

**Comment by BillWSY** [comment]
That's cool. It is the thing we should investigate carefully. It may or may not be the perfect fit at the end of the day, but at least we can learn a lot from it.

**Comment by zephirefaith** [comment]
Yeah, I feel like the less we re-invent the wheel the better it is. That also applies to learning how other frameworks work so that we can borrow something useful! Glad you think so 😄 !
